diff --git a/node_modules/nestjs-redis/dist/redis-core.module.js b/node_modules/nestjs-redis/dist/redis-core.module.js
index 3d1a982..ac2da9d 100644
--- a/node_modules/nestjs-redis/dist/redis-core.module.js
+++ b/node_modules/nestjs-redis/dist/redis-core.module.js
@@ -41,7 +41,7 @@ let RedisCoreModule = RedisCoreModule_1 = class RedisCoreModule {
             exports: [redis_service_1.RedisService],
         };
     }
-    onModuleDestroy() {
+    beforeApplicationShutdown() {
         const closeConnection = ({ clients, defaultKey }) => options => {
             const name = options.name || defaultKey;
             const client = clients.get(name);
