diff --git a/node_modules/wechaty/src/io-peer/json-rpc-peer.d.ts b/node_modules/wechaty/src/io-peer/json-rpc-peer.d.ts
index 18f9205..23ccaba 100644
--- a/node_modules/wechaty/src/io-peer/json-rpc-peer.d.ts
+++ b/node_modules/wechaty/src/io-peer/json-rpc-peer.d.ts
@@ -10,7 +10,7 @@ declare module 'json-rpc-peer' {
 
   export * from 'json-rpc-protocol'
 
-  declare module 'json-rpc-peer' {
+  module 'json-rpc-peer' {
     export class Peer extends EventEmitter implements NodeJS.WritableStream {
 
       constructor(onmessage?: (message: JsonRpcPayload, data: any) => Promise<any>)
@@ -58,8 +58,6 @@ declare module 'json-rpc-peer' {
       end(str: string, encoding?: string, cb?: () => void): void
 
     }
-
-    export default Peer
   }
 
 }
